__tests__/app.test.js

```diff
@@ -24,6 +24,36 @@ describe("nc-games app", () => {
         });
     });
   });
+  describe("GET /api/reviews", () => {
+    it("200 OK: returns an array of review objects under the key of 'reviews'", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toHaveLength(13);
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("owner");
+            expect(review).toHaveProperty("title");
+            expect(review).toHaveProperty("review_id");
+            expect(review).toHaveProperty("category");
```

Comment:
I would test that each review toMatchObject so you can test the datatypes as well. For example title: expect.any(String)